modules/scala-3-step-filter/src/test/scala/ch/epfl/scala/debugadapter/internal/stepfilter/ScalaStepFilterBridgeTests.scala

```diff
@@ -69,21 +69,21 @@ class Scala31PlusStepFilterBridgeTests extends ScalaStepFilterBridgeTests(ScalaV
     assert(findM("example.A").isDefined)
     assertEquals(
       formatM("example.A"),
-      Some("example.A.m(): String")
+      Some("m(): String")
```

Comment:
I think we still want the class name:

```suggestion
      Some("A.m(): String")
```

Same in all the other tests.